internal/cmd/cmd.go

```diff
@@ -27,7 +32,14 @@ func Run() {
 			zl.RunE(),
 			SyncFlagsCmdFunc,
 		),
+		SilenceErrors: true,
+		SilenceUsage:  true,
 	}
+	rootCmd.SetFlagErrorFunc(func(cmd *cobra.Command, err error) error {
+		cmd.Println(err)
+		cmd.Println(cmd.UsageString())
+		return errParsing
+	})
```

Comment:
We use the same pattern in SpiceDB. The idea is that any failure does not cause `zed` to output the help. Before if, say, a bulk import failed, it would output the error, and then the usage.

Now the help blob is only output if there is a parsing error (i.e. incorrect command/flags usage)